openapi/endpoints/cards/create-card-async.yaml

```diff
@@ -33,6 +33,18 @@ post:
 
         **FUNDING_SOURCE_INVALID**
         Funding source cannot be used with this card program.
+
+        **KYC_REQUIRED**
+        KYC profile is absent, doesn't have PASSED status, or doesn't match card program region. Use the spending prerequisites endpoint for more details.
```

Comment:
"KYC profile is absent" -> "KYC profile is missing" to use a more usual language

Reply:
doesn't -> does not
(contraction are used in the informal language